src/cluster/pager.rs

```diff
@@ -39,6 +39,50 @@ impl<
     }
   }
 
+
+  /// Java native driver also uses this technique to
+  /// perform stateless pagination
+  /// (https://docs.datastax.com/en/developer/java-driver/3.2/manual/paging/)
+  pub fn query_with_pager_state<Q>(&'a mut self, query: Q, state: Option<String>) -> Result<QueryPager<'a, Q, SessionPager<'a, M, S, T>>, error::Error>
```

Comment:
Please create a similar method for `ExecPager`

Reply:
I added the method for `ExecPager`.